inventory.yaml

```diff
@@ -0,0 +1,16 @@
+---
+groups:
+- name: ssh_nodes
+  nodes: []
+- name: winrm_nodes
+  nodes:
+  - name: jie1juz3iq95imb.delivery.puppetlabs.net
```

Comment:
Should this be committed at all or should inventory.yaml be in gitignore?

It looks like it's committing a secret.

Reply:
oops, forgot to add this to my .gitignore